documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -77,10 +78,10 @@ steps:
 
 | Parameter          | Description |
 | -------------------|-------------|
+| `neoHome`           | The path to the `neo-java-web-sdk` tool that is used for the deployment. |
```

Comment:
What is the reason for the re-ordering of the parameters?

Reply:
I have just thought it is easier to read if they are in the same order as in the example above. 
It could be cool if we could reference the "official" neoDeploy documentation here instead to avoid double maintenance.

Reply:
>I have just thought it is easier

At some point in time we agreed on having the parameters in alphabetical order, but as always when things are done manually, such agreements fail.

>It could be cool

agreed, but other topic